android/assets/jsons/Translations/NewGame,SaveGame,LoadGame,Options.json

```diff
@@ -522,54 +522,6 @@
 		Czech:"Doba počátku hry"
 }
 
-	"Ancient":{
```

Comment:
these word can't be used in game because there are some words like "ancient era" in tech.json.

Reply:
Yeah, this is kinda obsolete, I wonder if it's actually used anywhere?

Reply:
no, it occurs first in start game screen, and its back must + "era".

Reply:
So migrate this to "Ancient Era" etc.